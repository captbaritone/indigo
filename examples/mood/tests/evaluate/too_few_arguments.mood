fn sum(a: f64, b: f64): f64 {
    a + b
}

pub fn test(): f64 {
    sum(1_f64)
}