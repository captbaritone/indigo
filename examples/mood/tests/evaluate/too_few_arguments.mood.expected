-----------------
INPUT
----------------- 
fn sum(a: f64, b: f64): f64 {
    a + b
}

pub fn test(): f64 {
    sum(1_f64)
}
-----------------
OUTPUT
-----------------
Error: Too few arguments. Expected 2 but found 1.:
 --> too_few_arguments.mood:6:14

  | pub fn test(): f64 {
6 |     sum(1_f64)
  |              ^ "sum" requires 1 more argument.
  | }
